### PR TITLE
adding backend handler and converting manage listings from a tag to …

### PR DESCRIPTION
**Pull request by georgeteo**
New manage listings. 

Take a look

### CODE REVIEW

File: caravel/controllers/listings.py [hunk 1]
```diff
@@ -105,4 +105,10 @@ def create(keyword):
 @app.route('/logout')
 def logout():
     session.clear()
+    return redirect(url_for('index'))
+
+@app.route('/test')
+def test():
```
Comment (fatlotus) [inline caravel/controllers/listings.py]: Er, what? Remember, app versions all use the same shared key — so this gives someone admin access on all versions of the app!

**Comment by fatlotus** [comment]
Other than the /test note, looks good to me.

**Comment by georgeteo** [comment]
Yeah. Will get rid of it before the final merge.

**Comment by georgeteo** [comment]
Is fixed.

**Pull request merged**